tests/core/test_examples.py

```diff
@@ -23,16 +23,15 @@ async def test_moodbot_example(unpacked_trained_moodbot_path: Text):
 
 @pytest.mark.timeout(300)
 async def test_formbot_example(form_bot_agent: Agent):
-    async def mock_form_happy_path(
-        input_text: Text, output_text: Text, slot: Optional[Text] = None
-    ) -> None:
+    def response_for_slot(slot: Text) -> Dict[Text, Any]:
```

Comment:
fixed this test for you as it was a bit hairy

Reply:
Great, thanks!